doc/source/serve/model_composition.md

```diff
@@ -149,8 +150,8 @@ The downstream model has two methods:
 
 ### Advanced: Convert a DeploymentResponse to a Ray ObjectRef
 
-Under the hood, each `DeploymentResponse` corresponds to a Ray `ObjectRef` (or a `StreamingObjectRefGenerator` for streaming calls).
-In order to compose `DeploymentHandle` calls with Ray Actors or Tasks, you may want to resolve the response to its `ObjectRef`.
+Under the hood, each `DeploymentResponse` corresponds to a Ray `ObjectRef`, or a `StreamingObjectRefGenerator` for streaming calls.
+To compose `DeploymentHandle` calls with Ray Actors or Tasks, you may want to resolve the response to its `ObjectRef`.
 For this purpose, you can use the {mod}`DeploymentResponse._to_object_ref <ray.serve.handle.DeploymentResponse._to_object_ref>` and {mod}`DeploymentResponse._to_object_ref <ray.serve.handle.DeploymentResponse._to_object_ref_gen>` developer APIs.
```

Comment:
@edoakes, these two links are just broken.

Reply:
Hmm, yeah not ideal, I can make a follow-up PR to document the methods (I think we can whitelist them despite being underscore prefixed). For now we can just link to `DeploymentResponse`

Reply:
ack